packages/expo-stories/clients/app/ExpoStoryApp.tsx

```diff
@@ -0,0 +1,174 @@
+import { lightTheme, shadows, spacing } from '@expo/styleguide-native';
```

Comment:
Let's split this file a bit, shall we? 🤔 
I know the components aren't this complicated yet, but eventually they possibly would be 😅